.github/utils/patch_moment_locales.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2024 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+"""Patch Indico to include all moment.js locales.
+
+This updates the 'indico/web/client/js/jquery/index.js' file to
+include all moment.js locales from 'node_modules/moment/locale'.
+"""
+
+import difflib
+import re
+import sys
+from pathlib import Path
+
+import click
+from pygments import highlight
+from pygments.formatters.terminal256 import Terminal256Formatter
+from pygments.lexers.diff import DiffLexer
+
+
+START_MARKER = '// moment.js locales'
+END_MARKER = '// Last imported locale is used as fallback'
+
+
+def _format_import(locale):
+    return f"import 'moment/locale/{locale}';"
+
+
+def generate_locale_imports(indico_root):
+    locales = indico_root / 'node_modules' / 'moment' / 'locale'
+    for locale in sorted(locales.glob('*.js')):
+        if locale.stem != 'en-gb':
+            yield _format_import(locale.stem)
```

Comment:
I thought the last option we discussed was only using the locales that have been downloaded to indico (using the yaml file to lookup the corresponding moment locales)